web/src/shared/components/ToggleDarkMode.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
Should these components go in a Header folder?

Reply:
Donezo